contracts/Verifier.sol

```diff
@@ -0,0 +1,257 @@
+pragma solidity 0.5.2;
+pragma experimental ABIEncoderV2;
+
+import "./IEnforcer.sol";
+import "./IVerifier.sol";
+import "./IEthereumRuntime.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+//import "./MerkleProof.slb";
+//import "./Hash.slb";
+
+
+contract Verifier is Ownable {
+
+    struct Proofs {
+        bytes stackHash;
```

Comment:
Note: I use `bytes` instead of `bytes32` to save space if we don't need a hash.
But I yet have to check if that is really saving something 😄

Reply:
i think `bytes` uses at least 3 words (offset, length, data), while bytes32 is 2 word (length, data). i only know that about calldata thought.

Reply:
Yep, bytes32 is way cheaper 🔨